README.md

```diff
@@ -98,7 +118,11 @@ Run the following command to uninstall Cortex globally on your machine:
 
 ```
 # Uninstall globally using NPM
-npm uninstall -g @janhq/cortex
+npm uninstall -g cortexso
+
+# Uninstall using Brew
+brew uninstall cortexso
+brew untap janhq/cortexso
```

Comment:
@irfanpena please help separate npm and brew installation into 2 sections. I mean cortex installed by npm would not be uninstalled using brew, so better to not group by action.

Reply:
This is to make it concise. I have updated and included the uninstall into the quickstart section to separate it. Kindly check @louis-jan